virt/release_notes/virt-4-19-release-notes.adoc

```diff
@@ -151,6 +151,9 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 //CNV-52857 and CNV-56324
 * You can now xref:../../virt/vm_networking/virt-setting-interface-link-state.adoc#virt-setting-interface-link-state[manage the link state] of a primary or secondary virtual machine (VM) interface by using the {product-title} web console or the CLI.
 
+//CNV-58005
+* {VirtProductName} is now supported on link:https://learn.microsoft.com/en-us/azure/azure-boost/overview[Azure Boost], a set of features designed to enhance the performance and security of virtual machines.
```

Comment:
I don't understand what does ", a set of features designed to enhance the performance and security of virtual machines".
We should just state that OCP Virt is now supported on Azure Boost as Technology Preview.